src/views/JoinHouse.vue

```diff
@@ -82,7 +82,9 @@ methods: {
     joinHouse(){
         if(this.roomkey === 32){
             this.$store.dispatch("joinHouse", {email: this.$store.state.user.email, roomkey: this.roomkey})
-        } 
+        } else {
+            window.alert('Wrong House Key Input')
```

Comment:
Let's be super User Centered: "Your house key must be exactly 32 characters long"

Reply:
fixed